autoload/denops/_internal/server/proc.vim

```diff
@@ -164,8 +164,5 @@ augroup END
 
 call denops#_internal#conf#define('denops#_internal#server#proc#deno', g:denops#deno)
 call denops#_internal#conf#define('denops#_internal#server#proc#deno_args', filter([
-      \ '-q',
-      \ g:denops#type_check ? '' : '--no-check',
-      \ '--unstable',
-      \ '-A',
-      \], { _, v -> !empty(v) }))
+      \ g:denops#type_check ? '--check=all' : '',
+      \], { _, v -> !empty(v) }) + g:denops#server#deno_args)
```

Comment:
I think we should get rid of `g:denops#type_check` and add a `call denops#type_check()` function or something to call `deno check` instead. Because functions are generally easier to control (e.g. `call denops#type_check({plugin})` can perform a type check for a specific plugin or something).

Reply:
That sounds good, but it is out scope of this PR. I will remove `g:denops#type_check` and we should add the feature of type checking in other PR.

Reply:
Make sense.